lib/config.js

```diff
@@ -1,30 +1,38 @@
 'use strict';
 
 const fs = require('fs');
-const Path = require('path');
+const path = require('path');
 const os = require('os');
 
 class Config {
-  constructor (path) {
-    Object.defineProperty(this, 'configPath', { value: path });
+  constructor(configPath) {
+    Object.defineProperty(this, 'configPath', {
+      value: configPath
+    });
     this.load();
   }
 
   load() {
     this.hosts = {}
 
-    if (!fs.existsSync(this.configPath)) return;
+    if (!fs.existsSync(this.configPath)) {
```

Comment:
FWIW I think the one-line style is perfectly acceptable for control flow statements (pretend it's a return-if special form ;)) but I understand prettier/etc. auto-fixes this so meh.